packages/demo-site/scripts/build.js

```diff
@@ -11,7 +11,16 @@ const compileTemplate = async () => {
     const ejsTemplate = await fs.readFileSync(ejsFileDir, { encoding: 'utf8' })
 
     const template = ejs.compile(ejsTemplate)
-    const html = template({ API_KEY: config.API_KEY, CDN_URL: config.CDN_URL })
+
+    let CDN_URL = config.CDN_URL
+    const env = process.env.NODE_ENV
+
+    if (env !== 'DEV') {
+      CDN_URL = 'http://localhost:5000'
```

Comment:
Sorry, I am a bit confused by this - why are we serving CDN files from localhost in prod?

Reply:
It's a typos